test/doseCalc/test_BEDprojection.m

```diff
@@ -0,0 +1,57 @@
+function test_suite = test_BEDprojection
+
+test_functions = localfunctions();
+
+initTestSuite;
+
+function test_BEDprojectionConstructor
+bed = matRad_BEDProjection;
+assertTrue(isobject(bed));
+assertTrue(isa(bed, 'matRad_BEDProjection'));
+
+%test compute single scenario BED projection 
+function test_BED_computeSingleScenario
+dij.physicalDose{1} = 2;
+dij.mAlphaDose{1} = 1;
+dij.mSqrtBetaDose{1} = sqrt(0.05)*2;
+dij.ax{1} = 0.5;
+dij.bx{1} = 0.05;
+dij.doseGrid.numOfVoxels = 1;
+dij.ixDose{1} = 1;
```

Comment:
To generate these dummy dij's (and maybe also w) you could also write a function like helper_getTestDij(...). I have done this in other places, either as separate function in the test file (only functions with "test_" are added to the testsuite automatically) or as its own file if used by multiple functions.